secant/Features/App/Views/AppView.swift

```diff
@@ -18,17 +18,52 @@ struct AppView: View {
                     )
                 }
                 .navigationViewStyle(StackNavigationViewStyle())
+
             case .onboarding:
                 OnboardingScreen(
                     store: store.scope(
                         state: \.onboardingState,
                         action: AppAction.onboarding
                     )
                 )
+
             case .startup:
                 ZStack(alignment: .topTrailing) {
                     StartupView(sendAction: viewStore.send)
                 }
+
+            case .phraseValidation:
+                NavigationView {
+                    RecoveryPhraseBackupValidationView(
+                        store: store.scope(
```

Comment:
Same comment wit hate above conversation, it was marked as resolved but the formatting is still off below.

Reply:
Just bumping this one up